partner_communication/models/communication_job.py

```diff
@@ -781,8 +781,6 @@ def _print_report(self):
             state = "done"
             if job.need_call == "after_sending":
                 state = "call"
-            elif origin == "both_print":
-                state = "pending"
```

Comment:
The code is right and should not be changed. It's the view in xml that is wrong.

Reply:
I pushed a corrected version with this change. However I find it a bit weird because as far as I can see `both_print` is only set in this XML, and as such I don't see when the conditions related to `both_print` will be triggered if it is removed.

Reply:
Yes in fact we will need to put it back in the view in a correct way. But for now I think I will already merge this.